packages/stark-ui/src/modules/collapsible/components/_collapsible-theme.scss

```diff
@@ -0,0 +1,44 @@
+/* ============================================================================== */
+/*  S t a r k   C o l l a p s i b l e   -   T h e m e                             */
+/* ============================================================================== */
+/* stark-ui: src/modules/collapsible/components/_collapsible-theme.scss */
+
+.stark-collapsible {
+  .mat-expansion-panel {
+	box-shadow: $elevation-1;
+	&.mat-expanded {
+	  box-shadow: $elevation-16;
+	}
+	
+	.mat-expansion-panel-header {
+	  border-bottom: 1px solid mat-color($backgrounds, disabled-button);
+	  background-color: mat-color($backgrounds, background-light);
+	  &:not(.mat-expanded) {
+		&:hover {
+		  background-color: mat-color($backgrounds, background-light);
+		}
+	  }
+	  &.mat-expanded {
+		background-color: mat-color($mat-light-theme-background, card);
+	  }
+	}
+
+	.mat-expansion-panel-body {
+	  background-color: mat-color($backgrounds, background-light);
+	}
+
+	.mat-expansion-panel-header-description {
+	  color: mat-color(map-get($base-theme, primary-palette), 900);;
+	}
+
+	.mat-expansion-panel-header-title {
+	  color: mat-color(map-get($base-theme, primary-palette), 500);
+	}
+
+	.stark-collapsible-icon {
```

Comment:
Here I think we should add some global styles to icons, same way as the buttons `color="primary"` so we won't repeat this code each time we need a primary button.

@SuperITMan what do you think?

Reply:
My intention here was to put the icon on the same color as the title and match as much as possible the previous version of the collapsible.